ctx.go

```diff
@@ -1054,6 +1054,40 @@ func (c *Ctx) Redirect(location string, status ...int) error {
 	return nil
 }
 
+// Redirect to the URL derived from the specified path, with specified status.
+// If status is not specified, status defaults to 302 Found.
+func (c *Ctx) Redirect(location string, status ...int) error {
```

Comment:
could you explain the benefit of this wrapper function ?
we could use the Public function everywhere

Reply:
ah yes! will update function.